### ISSUE TITLE
Universal Import/Export tools.

### ISSUE THREAD

**Issue by Offline-R503B** [WolvenKit/WolvenKit#463]
Edited after rereview with @rfuzzo .

![image](https://user-images.githubusercontent.com/11785451/119777579-abcbe300-bec6-11eb-930c-498b6d41952c.png)

PE and I/E tool will be synchronized in itemsource. No Clicking buttons needed.

Image to indicate workflow (not completely correct as in this picture the synchronization isn't taken into account )  :

![image](https://user-images.githubusercontent.com/11785451/119777766-ea619d80-bec6-11eb-9502-4566274e10ef.png)
 

- [x] Create UI
- [ ] Create Logic

**Comment by ja-to** [comment]
This solution is significantly less efficient and robust compared to rfuzzo's existing asset pipeline. To demonstrate why, let's compare the workflow of importing a texture.

Existing workflow:

1) Export vanilla texture
2) Double click vanilla texture in project explorer
3) Edit and resave texture
4) Click Import button
5) Build project / Done

This proposal:

1) Export vanilla texture, get pop-up box to choose destination
2) Open Windows explorer and navigate to the texture
3) Edit and resave texture
4) Open Windows explorer and navigate to the texture, open WolvenKit, bring Windows explorer into focus, drag and drop into WolvenKit
5) Click Import Button
6) Open Windows explorer to drag and drop the imported file into the desired folder location
7) Build project / Done

In the best case scenario, it's significantly more tedious to minimize WolvenKit and navigate with Windows explorer. There are two extra common use cases that further diminish the efficiency.

Since most modders work iteratively, let's say we aren't happy with the results of our texture and we want to import once again. With the existing pipeline we open the .tga (or any raw) externally with WolvenKit project explorer, resave, click Import and done. Compare that to this proposal, where we navigate with Windows explorer, drag and drop, then have to manually place the texture into the project once again. After just one import, the process becomes extremely tedious and derails your work.

Secondly, imagine a modder who dumps 50 textures and uses AI upscaling for them. We now have 50 loose textures in one folder, and the author has to manually place each texture into the correct folder after each and every import.

There are also significant loses in usability. With rfuzzo's existing solution we can manage our raw source files efficiently inside the project instead of Windows explorer. We can duplicate, delete, open, etc. source files such as Blends, PS Documents, meshes, etc. directly inside WolvenKit. There are additional benefits such as including source files when using WolvenKit's project backup feature. Without the Raw directory we have no project management for our source files.

In summary, this solution is less efficient and eliminates project management for source files.

**Comment by ja-to** [comment]
Meshes and textures do not require any per-export settings.
